concepticondata/concepticon.tsv

```diff
@@ -3034,3 +3034,31 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3038	SAME-SEX YOUNGER SIBLING	Kinship	A person of the same sex who is born after another person who has the same parents.	Person/Thing	
 3039	DIFFERENT-SEX OLDER SIBLING	Kinship	A person of the different sex who is born before another person who has the same parents.	Person/Thing	
 3040	DIFFERENT-SEX YOUNGER SIBLING	Kinship	A person of the different sex who is born after another person who has the same parents.	Person/Thing	
+3041	PATERNAL AUNT (FATHER'S OLDER SISTER)	Kinship	The older sister of one's father.	Person/Thing	
+3042	MATERNAL AUNT (MOTHER'S OLDER SISTER)	Kinship	The older sister of one's mother.	Person/Thing	
+3043	MATERNAL AUNT (MOTHER'S YOUNGER SISTER)	Kinship	The younger sister of one's mother.	Person/Thing	
+3044	BOVINE	Animals	Member of the Bovinae subfamily, mostly pointing to bulls, cows and oxes.	Person/Thing	
+3045	COPPER (COLOR)	Sense perception	Having a red to yellow metallic color that resembles the metal copper.	Property	
+3046	FODDER	Food and drink	Feed for livestock, especially chopped hay or straw.	Person/Thing	
+3047	HOT SEASON	Time	Period of the year associated with hot or warm weather.	Person/Thing	
+3048	MOUTH HARP	Modern world	Lamellophone instrument consisting of a tongue or reed attached to a frame, placed in the performer's mouth and plucked; Jew's harp, jaw harp.	Person/Thing	
+3049	OIL (FROM ANIMALS)	Food and drink	Liquid hydrophobic substance from animals which is used for cooking.	Person/Thing	
+3050	SEEDLING	Agriculture and vegetation	A young plant produced from a seed.	Person/Thing	
+3051	REED PIPE	Modern world	Organ pipe that is sounded by a vibrating a metal tongue which rests agains a tunnel.	Person/Thing	
+3052	ERHU	Modern world	A two-stringed fiddle common in Eastern Asia; Chinese violin, Chinese two-stringed fiddle.	Person/Thing	
+3053	TIGHT	Basic actions and technology	Being not loose.	Property	
+3054	BEAR FRUIT	Agriculture and vegetation	To produce or yield fruits.	Action/Process	
+3055	SWINDLE	Law	To deceive or deprive by trickery and fraud; to cheat.	Action/Process	
+3056	ANIMAL (CLASSIFIER)	Quantity	Classifier for animals.	Classifier	
+3057	FLAIL	Basic actions and technology	A manual device used for beating or striking with, consisting of a free-swinging stick attached to a handle.	Person/Thing	
+3058	GRAZE	Animals	To feed on grasses and herbage in a field or on pastureland.	Action/Process	
+3059	MALARIA	The body	Infectious disease characterized by chills, fevers, and sweating, caused by a protozoan of the genus Plasmodium and transmitted by infected Anopheles mosquitos.	Person/Thing	
+3060	PHEASANT	Animals	Bird of the family Phasianidae.	Person/Thing	
+3061	PHLEGM	The body	Mucus secreted by the respiratory tract, particularly during respiratory infections, usually thick and sticky.	Person/Thing	
+3062	PIGEON	Animals	Bird of the family Columbidae.	Person/Thing	
```

Comment:
we have DOVE there, 1853, so please link to this one (similar to eggpland / aubergine)